qt/icons/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+set(SRC_FILES src/CharIconPainter.cpp src/CharIconEngine.cpp src/Icon.cpp)
+
+set(INC_FILES
+    inc/MantidQtIcons/CharIconPainter.h
+    inc/MantidQtIcons/CharIconEngine.h
+    inc/MantidQtIcons/Icon.h)
+
+qt4_add_resources(RES_FILES ${CMAKE_CURRENT_LIST_DIR}/resources/icons.qrc)
+
+if(ENABLE_MANTIDPLOT)
+  find_package(Qt4 COMPONENTS QtScript REQUIRED)
+  mtd_add_qt_library (TARGET_NAME MantidQtIcons
+                      QT_VERSION 4
+                      SRC ${SRC_FILES}
+                      NOMOC ${INC_FILES}
+                      DEFS
```

Comment:
It looks like the reason this works is that `mtd_add_qt_library` happens to use a variable named `RES_FILES` and sees this definition. The more modern way for us to do this is to add
```
RES ${CMAKE_CURRENT_LIST_DIR}/resources/icons.qrc
```
to the `mtd_add_qt_library` call here and remove the `qt4_add_resources` call above.